static/js/message_flags.js

```diff
@@ -6,12 +6,12 @@ import * as starred_messages from "./starred_messages";
 import * as ui from "./ui";
 import * as unread_ops from "./unread_ops";
 
-function send_flag_update(message, flag, op) {
+function send_flag_update_for_msgs(msg_ids, flag, op) {
```

Comment:
This should be `.._for_messages` -- we don't use `msgs` in the codebase.

Reply:
Thanks for pointing out. I'll keep that in mind 🙂